application/src/main/test/io/redlink/smarti/webservice/ConversationImporterTest.java

```diff
@@ -0,0 +1,124 @@
+import org.apache.http.entity.StringEntity;
+import org.apache.http.localserver.LocalTestServer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.redlink.smarti.services.ImporterService;
+import io.redlink.smarti.services.importer.ConversationSourceRocketChatJSONFile;
+import io.redlink.smarti.services.importer.ConversationSourceRocketChatMongoDB;
+import io.redlink.smarti.services.importer.ConversationSourceRocketChatWebservice;
+import io.redlink.smarti.services.importer.ConversationTargetSmarti;
+import io.redlink.smarti.services.importer.I_ConversationSource;
+import io.redlink.smarti.services.importer.I_ConversationSource.E_SourceType;
+import io.redlink.smarti.services.importer.I_ConversationTarget;
+import io.redlink.smarti.webservice.pojo.RocketFileConfig;
+import io.redlink.smarti.webservice.pojo.RocketMongoConfig;
+import io.redlink.smarti.webservice.pojo.RocketWebserviceConfig;
+
+/**
+ * 
+ * @author Ruediger Kurz (ruediger.kurz@deutschebahn.com)
+ * @since 03.08.2017
+ */
+public class ConversationImporterTest {
+
+	/** The database name of Rocket.Chat. */
+    private static final String ROCKETCHAT_MONGO_DB_NAME = "rocketchat";
+	
+	/** The name of the Rocket.Chat collection that holds the chat messages. */
+	private static final String ROCKETCHAT_MESSAGE_COLLECTION = "rocketchat_message";
+	
+	/** The name of the Rocket.Chat collection that holds the rooms.  */
+	private static final String ROCKETCHAT_ROOM_COLLECTION = "rocketchat_room";
+    
+	/** The host name of the Rocket.Chat MongoDB server. */
+	private static final String ROCKETCHAT_MONGO_HOST = "localhost";
+	
+	/** The port number of the Rocket.Chat MongoDB. */
+	private static final int ROCKETCHAT_MONGO_PORT = 27017;
+	
+	/** The name of a room field/property that is used to filter the rooms that should be exported from Rocket.Chat. */
+	private static final String ROCKETCHAT_FILTER_ROOM_FIELD_NAME = "expertise";
+	
+	/** The value of the room field that must match; a regex can be used. */
+	private static final String ROCKETCHAT_FILTER_ROOM_FIELD_VALUE = "assistify";
+
+	/** The absolute path to the Rocket.Chat JSON export. */
+	private static final String FILE_IMPORT_SOURCE = "file:///Users/rudiger/projects/smarti/application/src/test/resources/export-sapsus-faqs.json";
+	
+	/** The id of the client that is used to identify the knowledge domain in Smarti. */
+	private static final String SMARTI_CLIENT_ID = "test.assistify.de";
+	
+    private LocalTestServer server = new LocalTestServer(null, null);
+
+    @Before
+    public void setUp() throws Exception {
+        server.start();
+        server.register("*", (httpRequest, httpResponse, httpContext) ->
+                httpResponse.setEntity(new StringEntity("foobar")));
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        server.stop();
+    }
+    
+	@Test
+	public void testJSONFile() throws ClientProtocolException, IOException, Exception {
```

Comment:
There is not test right ;) So no assertions oa. If you want to do a nearly full integration which is still testable you could mock the conversation repository. Or test the units and mock the services. e.g. with Mockito.